### PR TITLE
PHP 7.1, Travis stages, updated dependencies, phpstan

### PR DESCRIPTION
**Pull request by mabar**

### CODE REVIEW

File: src/PluggableConfigurator.php [hunk 1]
```diff
@@ -105,15 +81,15 @@ protected function compile(Configurator $configurator, Compiler $compiler)
...
-	 * @return void
+	 * @param array<int,mixed> $params
 	 */
-	protected function trigger($class, ...$params)
+	protected function trigger(string $class, ...$params): void
 	{
 		foreach ($this->plugins as $plugin) {
 			// Skip different plugin
-			if (!($plugin instanceof $class)) continue;
+			if (!($plugin instanceof $class)) {
```
Comment (f3l1x) [inline src/PluggableConfigurator.php]: Oneline please

File: src/ExtraConfigurator.php [hunk 2]
```diff
@@ -153,34 +145,31 @@ public static function parseAllEnvironmentParameters()
...
+	public static function parseDebugValue(string $debug)
 	{
 		$value = $debug;
 
 		if (strtolower($value) === 'true' || $value === '1') {
-			$debug = TRUE;
-		} else if (strtolower($value) === 'false' || $value === '0') {
-			$debug = FALSE;
+			$debug = true;
+		} else {
```
Comment (f3l1x) [inline src/ExtraConfigurator.php]: Wut?

File: src/ExtraConfigurator.php [hunk 3]
```diff
@@ -31,76 +31,68 @@ protected function getDefaultParameters()
...
 	 * Parse environment parameters with NETTE__ prefix
 	 *
 	 * @return array
 	 */
-	public static function parseEnvironmentParameters()
+	public static function parseEnvironmentParameters(): array
 	{
 		$map = function (&$array, array $keys, $value) use (&$map) {
-			if (count($keys) <= 0) return $value;
+			if (count($keys) <= 0) {
```
Comment (f3l1x) [inline src/ExtraConfigurator.php]: Oneline please

File: src/ExtraConfigurator.php [hunk 4]
```diff
@@ -31,76 +31,68 @@ protected function getDefaultParameters()
...
-		if (!$fileName && !$appDir) return;
+		$appDir = $this->parameters['appDir'] ?? null;
+		if ($fileName === null && $appDir === null) {
+			return;
+		}
 
 		// Try to load file
 		$content = @file_get_contents($fileName ?: $appDir . '/../.debug');
-		if ($content === FALSE) return;
+		if ($content === false) {
```
Comment (f3l1x) [inline src/ExtraConfigurator.php]: Oneline please

File: src/ExtraConfigurator.php [hunk 5]
```diff
@@ -31,76 +31,68 @@ protected function getDefaultParameters()
...
-	 * @return void
-	 */
-	public function setFileDebugMode($fileName = NULL)
+	public function setFileDebugMode(?string $fileName = null): void
 	{
 		// Given file name or default file path
-		$appDir = $this->parameters['appDir'] ? $this->parameters['appDir'] : NULL;
-		if (!$fileName && !$appDir) return;
+		$appDir = $this->parameters['appDir'] ?? null;
+		if ($fileName === null && $appDir === null) {
```
Comment (f3l1x) [inline src/ExtraConfigurator.php]: Oneline please

File: .editorconfig [hunk 6]
```diff
@@ -1,26 +1,16 @@
...
-indent_style = tab
-indent_size = 4
-
-# NEON
-[*.neon]
-charset = utf-8
+trim_trailing_whitespace = true
 indent_style = tab
-indent_size = 4
+indent_size = tab
```
Comment (f3l1x) [inline .editorconfig:11]: Make exception for markdown, it cannot be tabbed (due to Github red blackground)

**Review by f3l1x — CHANGES_REQUESTED** [review]

File: src/ExtraConfigurator.php (hunk 2 above)
Comment (mabar) [reply src/ExtraConfigurator.php]: Phpstorm did it without warning. I don't understand why, my configuration should only convert `else if` to `elseif`

**Pull request merged**